File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 11/IWillShower.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_11
{
	internal class IWillShower : StoryChoice
	{
		internal IWillShower() : base(StoryChoice.GetLocalizedString("ch11IWillShower"), 11125)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch11ActedInSarahRoom = ActivityInSarahRoom.Showered;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_11/IWillShower.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_11;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.ActivityInSarahRoom;

/**
 * Created by christopherraleigh on 2016-05-22.
 */
public class IWillShower extends StoryChoice {
    public IWillShower() {
        super(R.string.ch11IWillShower, 11125);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh11ActedInSarahRoom(ActivityInSarahRoom.Showered);
        super.select();
    }
}
